modules/aioseop_sitemap.php

```diff
@@ -403,7 +446,175 @@ public function __construct() {
 			// TODO is this required for dynamic sitemap?
 			add_action( 'transition_post_status', array( $this, 'update_sitemap_from_posts' ), 10, 3 );
 			add_action( 'after_doing_aioseop_updates', array( $this, 'scan_sitemaps' ) );
+			add_action( 'after_doing_aioseop_updates', array( $this, 'upgrade_excluded_categories_to_excluded_terms' ) );
 			add_action( 'all_admin_notices', array( $this, 'sitemap_notices' ) );
+			add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ), 20 );
```

Comment:
Unsure if you are aware, but adding this hook is usually already handled by the parent class; when using the same function/method name.

Reply:
have renamed the function

Reply:
I tested operations with the parent method, and this can be removed. This works with the parent class admin_enqueue_scripts as intended.

I've uploaded a commit showing the changes https://github.com/EkoJR/all-in-one-seo-pack/commit/998cd74b2a6ba25dfdf45b63f4401c8ec61d8019